### ISSUE TITLE
weight norm

### ISSUE THREAD

**Issue by icy0101** [taki0112/TripleGAN-Tensorflow#4]
I thk u can do as what https://github.com/JayceeLee/Adversarial-Dropout has done. But still can not reproduce your result from the training graph u provided.

**Comment by icy0101** [comment]
sorry, i find that is not actually weight norm, so i try to use weightnorm in https://github.com/CuriousAI/mean-teacher/blob/master/tensorflow/mean_teacher/weight_norm.py

which will leads to a better result for a single classifier.
